src/index.html

```diff
@@ -6,10 +6,14 @@
     <script type="importmap">
       {
         "imports": {
-          "forma-embedded-view-sdk/auto": "https://cdn.skypack.dev/forma-embedded-view-sdk@^0.3/auto"
+          "forma-embedded-view-sdk/auto": "https://esm.sh/forma-embedded-view-sdk@0.x/auto"
         }
       }
     </script>
```

Comment:
I would prefer moving this import into the `src/sdk.js` as that would be consistent with how we recommend using this package: https://app.autodeskforma.com/forma-embedded-view-sdk/docs/index.html

Reply:
It's better to recommend using import maps as we avoid the situation that you use different versions of the package. Keep in mind that the example provided in the SDK is the least required to do anything - although we might recommend a importmap there

Reply:
If we are planning to introduce the `importmap`, then we should do that in both repositories! Is that mandatory, I mean, there could exist two different valid ways to achieve the same objective, Right?